internal/core/metadata/rest_addressable_test.go

```diff
@@ -674,6 +698,7 @@ func createAddressables(howMany int) []contract.Addressable {
 }
 
 func createMockAddressLoaderStringArg(howMany int, methodName string, arg string) interfaces.DBClient {
+
```

Comment:
Commented here; applies elsewhere.

Empty line after a function/method declaration should only be inserted after a multi-line (i.e. folded) function/method declaration.  This is a one line declaration.  Remove the empty line.